support-frontend/assets/pages/[countryGroupId]/checkout/helpers/getProductFields.ts

```diff
@@ -34,6 +34,11 @@ export const getProductFields = ({
 		billingPeriod: 'Monthly',
 	};
 
+	const fulfilmentOptions = getFulfilmentOptionFromProductKey(productKey);
+	const productOptions = getProductOptionFromProductAndRatePlan(
```

Comment:
Should both these `const`s be singular (to match the function name being called)? I think we're only getting back one fulfilmentOption and one productOption in each case?